test/app-luatest/console_debugger_session_test.lua

```diff
@@ -79,17 +124,31 @@ local function get_key_arg_pair(cmd)
 end
 
 local g = t.group('debug', {
-    {sequence = 'full scenario'},
-    {sequence = 'successful breakpoints additions'},
-    {sequence = 'failed breakpoints additions'},
-    {sequence = 'successful breakpoints removals'},
-    {sequence = 'failed breakpoints removals'},
+    {sequence = 'full'},
+    {sequence = 'bps_ok'},
+    {sequence = 'bps_fail'},
+    {sequence = 'bpd_ok'},
+    {sequence = 'bpd_fail'},
```

Comment:
I spend a number of seconds to understand that `bpd_fail` and `bps_fail` are not the same. Please rename.
The same with `bps_ok`/`bpd_ok`.